tools/python_stage.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# copies rust sources into Python package
+# (but not target)
+# 
+# vendors dependencies
+
+# exit immediately upon failure, unset vars
+set -e -u
+
+function usage() {
+  echo "Usage: $(basename "$0") [-c]" >&2
+}
+
+function log() {
+  local FORMAT="$1"
+  shift
+  local MESSAGE
+  MESSAGE=$(printf "$FORMAT" "$@")
+  echo "$MESSAGE" >&2
+}
+
+function run() {
+  local ARGS=("$@")
+  log "$ %s" "${ARGS[*]}"
+  eval "${ARGS[@]}"
+}
+
+function clean() {
+  log "***** CLEAN *****"
+
+  if [ -f "python/src/opendp/rust/Cargo.toml" ]; then
+    run cargo clean --manifest-path python/src/opendp/rust/Cargo.toml
+  fi
+  # "-X" removes only git-ignored files: Other local changes are preserved.
+  run git clean -x --force python
```

Comment:
```suggestion
  # "-X" removes only git-ignored files: Other local changes are preserved.
  # "-x" (lowercase) instead .... which we want because ....
  run git clean -x --force python
```
Just glancing at this, I'm not sure if the lowercase `-x` is correct.

Reply:
Copied from R, I adjusted the comment there as well.